docs/en_US/install/3ds.md

```diff
@@ -63,11 +80,9 @@ Your selection persists between reboots.
 
 ## Signing into your PNID
 
-To fully enjoy Pretendo Network, you'll need to sign into your PNID. It's not required for most things, but it's worth doing in case you need it in the future. If you haven't already, please [create a Pretendo account](/account) on your computer or phone before following the rest of the guide.
-
-On your 3DS, open up System Settings and tap on Nintendo Network ID Settings. It'll take a while to load, but once you do, follow along until the part where you get asked whether you want to link your account. Select `Yes` and use the credentials you used to create your Pretendo account. Once you've done that, continue normally and you should be all set up.
+The 3DS does not rely on NNIDs for the vast majority of it's game servers. Because of this, using a PNID is also not required for most games<sup><a>[[1]](#footnote-1)</a></sup>.
 
-Congratulations! You can now use Pretendo Network and enjoy 3DS games online even after April 2024.
+Setting up a PNID on the 3DS is the same as setting up a NNID. You may either create the PNID on your console, or register from an account [on our website](/account/register) and link it to your console at a later date
```

Comment:
I would recommend using the 3DS to create the account since account details can't be modified from the website yet

Reply:
How does this sound?

```suggestion
Setting up a PNID on the 3DS is the same as setting up a NNID. You may either create the PNID on your console, or register from an account [on our website](/account/register) and link it to your console at a later date

It is recommended to register the PNID on your device at this time, as registering on the website does not currently allow you to change your user data
```

Reply:
Looks good! :+1: